python/pyspark/sql/dataframe.py

```diff
@@ -1311,6 +1329,28 @@ def fill(self, value, subset=None):
     fill.__doc__ = DataFrame.fillna.__doc__
 
 
+class DataFrameStatFunctions(object):
+    """Functionality for statistic functions with :class:`DataFrame`.
+    """
+
+    def __init__(self, df):
+        self.df = df
+
+    def cov(self, col1, col2):
```

Comment:
I noticed how we do it, but that doc contains `is alias for doc.stat.cov`. Wouldn't it be weird to have that in this doc as well? If I remove that line, it would make sense.